js/background.js

```diff
@@ -186,14 +316,15 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) =>
 // This function updates the extension's toolbar icon
 function UpdateTheme()
 {
+	var collectionsLength=Object.keys(collections).length
```

Comment:
Why can't we just use `collections.length`?

Reply:
Because the "length" property does not exist for JS objects, unfortunately